docs/source/concept_models/vivarium_swissre_cervicalcancer/cervical_cancer_concept_model.rst

```diff
@@ -397,7 +397,7 @@ Cervical cancer screening algorithm was determined by three variables
      - Specificity
    * - Cytology plus HPV test
      - HPV+: 76.7%
-     - HPV-: TBD
+     - HPV-: 94.1%
```

Comment:
Thanks for getting this value in so quickly Yaqi! Could you please add a note that this value is HPV test specificity for HPV test alone as a proxy for HPV test specificity for cotest as well as cite the paper - thanks!!

Reply:
okay, will do